src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Hello from './components/Hello/Hello.jsx';
```

Comment:
add `index.js` file inside of `Hello` folder and export `Hello.jsx`. So, after that change this import to `import Hello from './components/Hello`